SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -1503,7 +1503,9 @@ FRPCErrorInfo USpatialReceiver::ApplyRPC(const FPendingRPCParams& Params)
 	const float TimeDiff = (FDateTime::Now() - Params.Timestamp).GetTotalSeconds();
 	if (GetDefault<USpatialGDKSettings>()->QueuedIncomingRPCWaitTime < TimeDiff)
 	{
-		UE_LOG(LogSpatialReceiver, Warning, TEXT("Executing RPC %s::%s with unresolved references after %f seconds of queueing"), *TargetObjectWeakPtr->GetName(), *Function->GetName(), TimeDiff);
+		if ((Function->SpatialFunctionFlags & SPATIALFUNC_AllowUnresolvedParameters) == 0) {
```

Comment:
```suggestion
		if ((Function->SpatialFunctionFlags & SPATIALFUNC_AllowUnresolvedParameters) == 0)
		{
```

Reply:
We might want to consider not even waiting to apply the RPC if this flag is true

Reply:
Agreed

Reply:
Oh, I remember now - I was unsure of the behaviour here. I suspected the wait time might be to wait for parameters to resolve in some way? (or why else do we wait here)

Reply:
Okay I agree, lets keep the wait time in and make that customisable in a follow up PR